kanalony-storage-api/src/main/scala/ConnectorFactory.scala

```diff
@@ -0,0 +1,10 @@
+package kanalony.storage.api
+
+import com.websudos.phantom.connectors.{ContactPoint, KeySpaceDef}
+
+/**
+ * Created by elad.benedict on 2/7/2016.
+ */
+object ConnectorFactory extends IConnectorFactory {
+  override val connector : KeySpaceDef = ContactPoint.local.keySpace("my_keyspace")
```

Comment:
keyspace should be defined per table (although in the first phase all tables will be on the same keyspace)